cmd/kured/main.go

```diff
@@ -627,13 +645,17 @@ func rebootAsRequired(nodeID string, rebootCommand []string, sentinelCommand []s
 			}
 		}
 
-		if !acquire(lock, &nodeMeta, TTL) {
+		if !holding(lock, &nodeMeta) && !acquire(lock, &nodeMeta, TTL) {
```

Comment:
not familiar with these lockign fucntkions but aquire doesn't return true if you are already holding? might be nice to extend the ttl?

Reply:
The current `acquire` func returns `false` if you're currently holding the lock. Didn't want to change that logic in case it's needed elsewehere.